### PR TITLE
[Docs] Many small tweaks

### PR DESCRIPTION
**Pull request by davidnuzik**
- Capitalize rio > Rio where appropriate
- Capitalize kubernetes > Kubernetes where appropraite
- Capitalize docker > Docker where appropriate
- Fix capitalization of github > GitHub where appropriate
- Fix areas were articles were not utilized ("a", "an", "the") where appropriate
- Various grammar tweaks and nitpicks

### CODE REVIEW

File: docs/autoscaling.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ $ rio run --scale 1-10 --concurrency 20 -p 8080 strongmonkey1992/autoscale:v0
 $ rio run --scale 0-10 -p 8080 strongmonkey1992/autoscale:v0
 ```
 
-To put load the following example use [hey](https://github.com/rakyll/hey):
+To test, put load the service. Use [hey](https://github.com/rakyll/hey):
```
Comment (cbron) [inline docs/autoscaling.md]:
```suggestion
To test putting load on the service use [hey](https://github.com/rakyll/hey):
```

File: docs/cli-reference.md [hunk 2]
```diff
@@ -244,7 +244,7 @@ rio export --riofile namespace/default
 
 ## image
 
-List images built from local registry
+List images built from a local registry
```
Comment (cbron) [inline docs/cli-reference.md]: Either use `the` or nothing, this means specifically the rio registry.

File: docs/autoscaling.md (hunk 1 above)
Comment (davidnuzik) [reply docs/autoscaling.md]: Addressed via https://github.com/rancher/rio/pull/830/commits/528e07d205269d9d06d7c76073eb296800f2f2d2

File: docs/cli-reference.md (hunk 2 above)
Comment (davidnuzik) [reply docs/cli-reference.md]: Addressed via https://github.com/rancher/rio/pull/830/commits/441b69f05001c62f12d6a95ddeeed81e46b44a06

**Pull request merged**